openshift/system/Dockerfile

```diff
@@ -27,11 +23,16 @@ RUN yum-config-manager --save --setopt=cbs.centos.org_repos_sclo7-rh-ruby26-rh-c
               postgresql10 postgresql10-devel postgresql10-libs \
               file \
               rh-nodejs12 \
-    && rpm -i /tmp/sphinxsearch.rpm \
-    && rm /tmp/*.rpm \
     && yum install -y epel-release \
     && yum -y clean all
 
+# Install sphinx search
+ARG SPHINX_VERSION=3.4.1
+COPY openshift/system/sphinx-${SPHINX_VERSION}-efbcc65-linux-amd64.tar.gz /tmp/sphinxsearch.tar.gz
+RUN tar zxf /tmp/sphinxsearch.tar.gz -C /tmp \
```

Comment:
Worth replacing this with a curl/wget command that downloads the package at every build, to avoid hosting the 20MB file in the code base? Granted it won't ever be yanked from source of course.

Reply:
Done.